web/blog/tenzir-node-v4.26/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Tenzir Node v4.26: Security Lake Integration"
+slug: tenzir-node-v4.26
+authors: [lava]
+date: 2025-01-08
+tags: [release, node]
+comments: true
+---
+
+# Tenzir Node v4.26: Security Lake Integration
+
+Tenzir Node v4.26 continues to leverage our native Parquet and OCSF
+capabilities by adding a new operator to write data directly to
+Amazon Security Lake.
+
+![Tenzir Node v4.26](tenzir-node-v4.26.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.26.0
+
+<!-- truncate -->
+
+## Amazon Security Lake
+
+The new `to_asl` operator allows users to write data directly to the
+Amazon Security Lake:
+
+```tql
+let $s3_uri = "s3://aws-security-data-lake-eu-west-2-lake-abcdefghijklmnopqrstuvwxyz1234/ext/tenzir_network_activity/"
+ 
+export
+where @name == "ocsf.network_activity"
+to_asl $s3_uri,
+  region="eu-west-2",
+  accountId="123456789012"
+```
+
+For more information, look at the [Integrations](/next/integrations/amazon/security-lake)
+page in our documentation.
+
+### Output Changes
+
+With this version, we made some changes in the way we render timestamps
+in order to make it easier for downstream tooling to work with the data
+provided by a Tenzir Node.
+
+:::warning Potentially Breaking Change
+
+This may break automations that parse the JSON output of a Tenzir Pipeline.
+:::
+
+### TQL Output
+
+By default, the node now outputs pipeline results as TQL rather than JSON.
+
+TQL is a superset of JSON, but it includes support for native IP, subnet,
+timestamp and duration types which were formerly rendered as strings.
+
+Additionally, TQL also supports and outputs trailing commas by default.
+
+To restore the previous behavior, use the `write_json` sink, or set
+a default sink to revert it globally:
+
+```env
+TENZIR_DEFAULT_SINK=write_json
```

Comment:
I'm curious: how does this interact with `TENZIR_EXEC__IMPLICIT_EVENTS_SINK="write_json | save_stdout"`, which has existed for a long time now?

Reply:
It doesn't, there was an implicit `FIXME: look up correct option name` here that I didn't write down and thus forgot to change before opening the PR. Fixed now.